presto-main/src/main/resources/webapp/query.html

```diff
@@ -62,6 +62,9 @@
         <dt>Query State</dt>
         <dd id="queryState"></dd>
 
+        <dt>Error Code</dt>
```

Comment:
I tried this out and it looks weird having them here. Put them below between "Kill" and "Message".